localmodel.Dockerfile

```diff
@@ -26,4 +27,6 @@ RUN /opt/app-root/src/go/bin/go-licenses save --save_path third_party/library ./
 FROM gcr.io/distroless/static:nonroot
 COPY --from=builder /go/src/github.com/kserve/kserve/third_party /third_party
 COPY --from=builder /go/src/github.com/kserve/kserve/localmodel-manager /manager
+USER 1000:1000
```

Comment:
also here, ubi for final image?

Reply:
same reply as the other comment about ubi